### DISCUSSION TITLE
24h Timeformat?

### DISCUSSION THREAD

**Discussion by ezadoo** [lukevella/rallly#121] [Ideas]
I‘ve looked in the issues and it seems that there isn’t an option to display the time in the 24h timeformat instead of the AM/PM format.

Is there a chance that something like that will be implemented?
It would be great, because I think the 24h system is much more used, at least in europe.

**Comment by lukevella** [comment]
Hi @ezadoo, that's a fair point. I'll open an issue for this and try to address it soon.
